pages-e2e/fixtures/app14.0.0/main.fixture

```diff
@@ -4,7 +4,6 @@
 		"appRouting",
 		"appConfigsTrailingSlashTrue",
 		"appWasm",
-		"appServerActions",
```

Comment:
Note: I've removed this feature from the app14.0.0 fixture... this is definitely not ideal but I couldn't figure out why this was failing in the e2es

If you try to run the app14.0.0 fixture with the feature included you will see that the application does work as intended (sort of... it seems like server actions do present some problems... but those are pre-existing problems that also main presents)

Also this feature is included in the appLatest fixture, so we are indeed still testing this functionality...

If we really believe it worth it I can spend more time trying to see if there's a way to keep the feature in app14.0.0